backup_swift/tasks.py

```diff
@@ -47,6 +47,7 @@ def do_backup_swift():
     """
     Perform full swift backup sequence.
     """
+    logger.warning('SWIFT storage is not maintained and will be dropped in a future version.')
```

Comment:
nit: I think there is a specific system for this in python:
``
import warnings
warnings.warn(message, DeprecationWarning)
```

Reply:
This warning doesn't use the module's logging system, so I decided to stick with a warning.